Doc/library/operator.rst

```diff
@@ -314,6 +314,8 @@ expect a function argument.
    method.  Dictionaries accept any hashable value.  Lists, tuples, and
    strings accept an index or a slice:
 
+      >>> itemgetter('name')({'name': 'tu', 'age': 18})
```

Comment:
Actually, this makes the example exactly parallel the paragraph it follows, showing the dictionary example first, then the index, and then the slice, in the order they are mentioned in the text.  Breaking the paragraph up with the example would not improve the flow, in my opinion.  The fact that dictionary is in a separate sentence doesn't that much, I think.